rpc_jobs/leapfrog_multi_node_aio.yml

```diff
@@ -0,0 +1,136 @@
+- project:
```

Comment:
This might be better as an addition to the exiting multi_node_aio.yml file instead of a separate one so that it's closer to what we do for AIO leapfrogs.